### PR TITLE
Config update

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -3,27 +3,25 @@
...
-    "typings/main.d.ts",
-    "node_modules/typescript/lib/lib.core.es6.d.ts"
+  "include": [
+    "src/**/*.ts"
   ],
-  "files": [],
-  "atom": {
-    "rewriteTsconfig": true
-  }
+  "exclude": [
```
Comment (rictic) [inline tsconfig.json]: This isn't strictly speaking necessary, as it's not in the include section. IMO, drop.

File: package.template.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "lint": "gulp lint",
+    "test": "npm run clean && npm run build && npm run lint && mocha",
+    "test:watch": "watchy -w src/ -- npm test --loglevel=silent",
+    "format": "find src test | grep '\\.js$\\|\\.ts$' | xargs ./node_modules/.bin/clang-format --style=file -i",
+    "prepublish": "tsc && npm test"
+  },
+  "author": "The Polymer Project Authors",
+  "license": "BSD-3-Clause",
+  "devDependencies": {
+    "clang-format": "=1.0.45",
```
Comment (rictic) [inline package.template.json:18]: Most of these deps should instead just be a dep on tools-common yeah?

**Review by rictic — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (justinfagnani) [reply tsconfig.json]: Done

File: package.template.json (hunk 2 above)
Comment (justinfagnani) [reply package.template.json:18]: Done

**Comment by justinfagnani** [comment]
PTAL

**Review by rictic — APPROVED** [review]

**Pull request merged**